explorer/src/components/BondBreakdown.tsx

```diff
@@ -11,103 +11,144 @@ import Paper from '@mui/material/Paper';
+
             setBonds({
-                delegations,
-                pledges,
-                bondsTotal,
-                denom: thisMixnode.total_delegation.denom.toUpperCase()
+                delegations: decimalisedDelegations,
+                pledges: decimalisedPledges,
+                bondsTotal: bondsTotal,
+                hasLoaded: true,
             });
         }
     }, [mixnodeDetailInfo]);
 
-    return (
-        <>
-            <TableContainer component={Paper}>
-                <Table sx={{ minWidth: 650 }} aria-label='bond breakdown totals'>
-                    <TableBody>
-                        <TableRow sx={matches ? { minWidth: '70vw' } : null}>
-                            <TableCell
-                                sx={{
-                                    fontWeight: 'bold',
-                                    width: matches ? '90px' : 'auto',
-                                }}
-                                align='left'
-                            >
-                                Bond total
-                            </TableCell>
-                            <TableCell align='left'>
-                                {bonds.bondsTotal}{bonds.denom}
-                            </TableCell>
-                        </TableRow>
-                        <TableRow>
-                            <TableCell
-                                sx={{
-                                    width: matches ? '90px' : 'auto',
-                                }}
-                                align='left'
-                            >
-                                Pledge total
-                            </TableCell>
-                            <TableCell align='left'>
-                                {bonds.pledges}{bonds.denom}
-                            </TableCell>
-                        </TableRow>
-                        <TableRow>
-                            <TableCell
-                                sx={{
-                                    width: matches ? '90px' : 'auto',
-                                }}
-                                align='left'
-                            >
-                                Delegation total
-                            </TableCell>
-                            <TableCell align='left'>
-                                {bonds.delegations}{bonds.denom}
-                            </TableCell>
-                        </TableRow>
-                    </TableBody>
-                </Table>
+    React.useEffect(() => {
+        const hasError = mixnodeDetailInfo?.error || delegations?.error ? true : false;
+        const hasAllMixnodeInfo = mixnodeDetailInfo?.data !== undefined && mixnodeDetailInfo?.data[0].mix_node ? true : false;
+        const hasAllDelegationsInfo = delegations?.data !== undefined && delegations?.data.length ? true : false;
+        const hasAllData = !hasError && hasAllMixnodeInfo && hasAllDelegationsInfo ? true : false;
```

Comment:
```suggestion
        const hasError = Boolean(mixnodeDetailInfo?.error || delegations?.error);
        const hasAllMixnodeInfo = Boolean(mixnodeDetailInfo?.data !== undefined && mixnodeDetailInfo?.data[0].mix_node);
        const hasAllDelegationsInfo = Boolean(delegations?.data !== undefined && delegations?.data.length);
        const hasAllData = Boolean(!hasError && hasAllMixnodeInfo && hasAllDelegationsInfo);
```

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean and:

```js
var x = Boolean(expression);     // use this...
var x = !!(expression);          // ...or this
```